### PR TITLE
Remove zendesk help and replace with "Contact us" email link

### LINKED ISSUE #1187: Remove Zendesk "Help" button
Replace the help button with a "Contact Us" link in the left nav. 

Zeplin: https://app.zeplin.io/project/5b181d3bf71408ae4d406b94/screen/5bd04211c19e0709b97fb780
InVision: https://projects.invisionapp.com/share/9EN908ZQPAX#/screens/327390107_Channel_Update_17_Oct

In the interests of time, this can be a simple mailto: link, to `OPEN_DISCUSSIONS_SUPPORT_EMAIL` address. 

When we have more time, we can display a contact form that integrates directly with the Zendesk API.

### PR DESCRIPTION
**Pull request by mbertrand**
#### Pre-Flight checklist

- [x] Screenshots and design review for any changes that affect layout or styling
  - [x] Desktop screenshots 
  - [x] Mobile width screenshots 
  - [x] tag @ferdi or @pdpinch for review  

#### What are the relevant tickets?
Closes #1187 

#### What's this PR do?
- Removes the zendesk help button
- Adds a 'Contact us' email link

#### How should this be manually tested?
View any page.  There should no longer be a 'Help' overlay button near the the botton right corner.  There should be a 'Contact us' link just above 'Community guidelines' on the left sidebar.  Clicking it should open up a new email addressed to the `OPEN_DISCUSSIONS_SUPPORT_EMAIL` address.

screen shot 2018-11-19 at 3 21 04 pm
screen shot 2018-11-19 at 3 20 50 pm

### CODE REVIEW

**Comment by pdpinch** [comment]
Thanks Matt. I love deleting code!

What's the 2nd screenshot? Should I be concerned about the lack of logo & address?

**Comment by mbertrand** [comment]
@pdpinch second screenshot is the desktop version, it's a partial shot, top and bottom cut off, just wanted to show that 'Contact us' is there.

**Comment by aliceriot** [comment]
I think there's an issue with the spacing. here's what the drawer looks like on master:

spacingbefore

and on this branch:

spacingafter

I think we had to do some manual positioning to work around a layout bug in the drawer, and the addition of another link is messing with the spacing.

File: static/js/components/Footer.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ import moment from "moment"
 
 const Footer = () => (
   <div className="footer">
+    <div className="row">
+      <a href={`mailto:${SETTINGS.support_email}`}>Contact us</a>
+    </div>
```
[the remark this answers is not in the document]
Comment (mbertrand) [reply static/js/components/Footer.js]: ✔️ good idea

**Pull request merged**